tests/h/services/search_index/_queue_test.py

```diff
@@ -350,7 +350,18 @@ def add_all(ids=annotation_ids, schedule_in=MINUS_5_MIN_IN_SECS, force=False):
 
 
 class TestCount:
-    def test_it(self, db_session, factories, now, queue):
+    @pytest.mark.parametrize(
+        "tags,expired,expected_result",
+        [
+            (None, False, 3),
+            (None, True, 1),
+            (["storage.create_annotation", "storage.update_annotation"], False, 2),
+            (["storage.create_annotation", "storage.update_annotation"], True, 1),
+        ],
+    )
+    def test_it(
+        self, db_session, factories, now, queue, tags, expired, expected_result
+    ):
```

Comment:
This is sadly a bit more obscure than the previous stuff, but you know what, it's metrics. Who cares. I think as long as we have some coverage it's grand.